src/gui/SearchDialog.cpp

```diff
@@ -112,23 +126,27 @@ void CompletionLabel::selectFirst()
 
 void CompletionLabel::updateText()
 {
-	QString newText;
+    QString newText;
+
+    // Regenerate the list with the selected item in bold (and italicize recent items)
+    for (int i=0;i<values.size();i++)
+    {
+        QString word = values[i];
```

Comment:
Use a more descriptive variable name than "word".  The expression `values[i]` is short enough it may be more readable without this variable.

Reply:
Done